bigbluebutton-html5/imports/api/captions/server/helpers.js

```diff
@@ -0,0 +1,56 @@
+import { Meteor } from 'meteor/meteor';
+import { hashFNV32a } from '/imports/api/common/server/helpers';
+import { check } from 'meteor/check';
+
+const CAPTIONS_CONFIG = Meteor.settings.public.captions;
+const LOCALES_URL = 'http://localhost:3000/html5client/locales';
```

Comment:
should at least use `basename/` from settings.yml. Still thinking how to make sure the url is correct. Maybe put the whole LocalesUrl in the configuration